packages/dashboard-frontend/src/store/index.ts

```diff
@@ -42,6 +43,7 @@ export interface AppState {
   clusterInfo: ClusterInfo.State;
   devWorkspaces: DevWorkspacesStore.State;
   devfileRegistries: DevfileRegistriesStore.State;
+  dwGitOauthConfig: GitOauthConfigStore.State;
```

Comment:
I'm curious why you added the `dw` prefix. Why not simply `gitOauthConfig`?

Reply:
I have fixed it.